src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', (ev) => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    const error = 'Time is over';
+
+    reject(error);
+  }, 3000);
+});
+
+function makeDiv(isThen) {
```

Comment:
```suggestion
function handleMessage(isThen) {
```

Reply:
Also, will be better if you pass messages and classes as props, not a boolean type